README.md

```diff
@@ -245,7 +245,7 @@ Alternatively you can also use a phpdoc for this:
 use Spatie\DataTransferObject\DataTransferObjectCollection;
 
 /**
- * @method PostData current
+ * @method \App\DTOs\PostData current
```

Comment:
@niektenhoopen I am pretty sure this is fine. It is used as an example in the README and necessary to achieve the auto completion mentioned. Note how these namespaces are used elsewhere in the existing README.

Reply:
Ah, I couldn't find it and I have now figured out why. In the existing readme it's `App\DataTransferObjects\PostData` instead of `App\DTOs\PostData`

Reply:
Yeah, event though it is longer it may be nice to have it be `App\DataTransferObjects\PostData` just to be consistent. I wouldn't let that keep the feature from coming through though. 👍🏼

Reply:
Let's see if this works. It would make it a one-click-change:
```suggestion
 * @method \App\DataTransferObjects\PostData current
```